pkgs/development/python-modules/ion-python/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, jsonconversion, six, pytest-runner, pytest }:
+
+buildPythonPackage rec {
+  pname = "ion-python";
```

Comment:
```suggestion
  pname = "amazon-ion";
```

Reply:
Is this the convention when the published package is named something like `amazon.ion`?

Reply:
We take the pip project name and normalize it. So `.` becomes `-`.